addons/event_sale/tests/test_event_sale.py

```diff
@@ -338,7 +338,13 @@ def _prepare_currency(self, currency_name):
 
         currency_VEF = _prepare_currency(self, 'VEF')
         currency_USD = _prepare_currency(self, 'USD')
-        self.env.user.company_id.currency_id = currency_USD
+
+        company = self.env['res.company'].create({
```

Comment:
Would name it `test_company`, highlighting the fact it is linked to the test. Does not change much here, but if someday this moves into a setup or equivalent to be used in several tests, it avoids having to grep "company" to see where it is used (grepping company might be complicated ^^ ).